app/controllers/application_controller.rb

```diff
@@ -2,4 +2,14 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  before_filter :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.for(:sign_up) { |u| u.permit(:name, :profile_name,:email, :personal_email, :job_title, :password) }
+    devise_parameter_sanitizer.for(:account_update) { |u| u.permit(:name, :profile_name,:email, :personal_email, :job_title,
+      :password, :current_password, :password_confirmation) }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Expression at 13, 61 should be on its own line.